packages/server/src/queue/queue.controller.ts

```diff
@@ -171,6 +171,12 @@ export class QueueController {
 
     queue.isDisabled = true;
 
+    // clear staff list
+    if (role !== Role.STUDENT) {
+      queue.staffList = [];
+      await queue.save();
+    }
```

Comment:
I don't think you need to check the role here because this endpoint already has role guards on it; also I dont think you need to call save() because that's being done at the end of this func already.